product/runtime/src/test/python/chaquopy/test/android/test_import.py

```diff
@@ -642,7 +642,7 @@ def check_iter_modules(mod, expected):
 
         # Check that a non-existent package returns an empty list.
         for path in ("somemissingpackage", "some/missing/package"):
-            mod_infos = list(pkgutil.iter_modules([path]))
+            mod_infos = list(pkgutil.iter_modules([f"{murmurhash.__path__[0]}/{path}"]))
             self.assertEqual([], mod_infos)
```

Comment:
This had the same problem as #924: it didn't reproduce the original problem, because it wasn't passing a path that actually led to an AssetFinder.